Clean up files & tests

RissyRan opened the pull request:
# Description

Clean up files & tests that was deleted but somehow re-added in later PRs:
* `configs/benchmark/jax` & `dags/benchmark/solutionsTeam_jax_npi.py` has already been removed in [PR 32](https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/32)
* `dags/xlml/solutionsteam_jax_integration.py` has already been renamed in [PR 38](https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/38)

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**Instruction and/or command lines to reproduce your tests:** ...

**List links for your tests (use go/shortn-gen for any internal link):** ...
Expect all checks pass.

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

gkroiz commented on dags/xlml/solutionsteam_jax_integration.py:
Hey @RissyRan want to double check here, when I looked at #38, the name here was changed to `solutionsteam_jax_latest_integration.py`. Here, we removed `latest`, is that intentional?

RissyRan commented:
Yes, it's expected. If you check another merged PR, changes are there as
well. We have latest and head in Jax, so I removed latest flag. Thanks for
double checking!

On Thu, Dec 7, 2023, 7:48 PM Gerson Kroiz ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> On dags/xlml/solutionsteam_jax_integration.py
> <https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/42#discussion_r1419897757>
> :
>
> Hey @RissyRan <https://github.com/RissyRan> want to double check here,
> when I looked at #38
> <https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/38>, the
> name here was changed to solutionsteam_jax_latest_integration.py. Here,
> we removed latest, is that intentional?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/42#pullrequestreview-1771491658>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE3Q5OUAFG4KRJDVGNUP7YDYIKEXZAVCNFSM6AAAAABAMB2JSWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTONZRGQ4TCNRVHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

gkroiz reviewed (approved):
LGTM
